package.json

```diff
@@ -6,7 +6,7 @@
   "main": "lib/main.js",
   "scripts": {
     "clean": "rm -rf node_modules package-lock.json",
-    "build-checkout": "npm install --prefix node_modules/checkout && npm run build --prefix node_modules/checkout",
+    "build-checkout": "npm install --prefix node_modules/checkout --force && npm run build --prefix node_modules/checkout",
```

Comment:
Added `--force` here to get around a version conflict message about eslint. Not sure if updating related dependencies would've been a better approach.